platforms/vmware/giantnetes/main.tf

```diff
@@ -76,6 +84,59 @@ module "vault" {
   depends_on = [module.nsxt]
 }
 
+module "flatcar_linux" {
```

Comment:
I'm a bit lost for what this is used. if you pass the template as variable to the module, then you are making the assumption that the template is already uploaded into VMware VSphere.

Reply:
good point - i meant to remove this (accidentally carried it over from another manifest)

Reply:
fixed.